core/src/main/java/pl/allegro/tech/embeddedelasticsearch/ElasticRestClient.java

```diff
@@ -180,6 +183,20 @@ void refresh() {
         }
     }
 
+    private boolean newESVersion() {
+        HttpGet request = new HttpGet(url("/"));
+        return httpClient.execute(request, response -> {
+            JsonNode jsonNode;
+            try {
+                jsonNode = OBJECT_MAPPER.readTree(readBodySafely(response));
+            } catch (IOException e) {
+                return false;
+            }
+            String esV = jsonNode.get("version").get("number").asText();
+            return Integer.parseInt(esV.substring(0,1)) >= 7; //if version is 7 and above
```

Comment:
What happens with elasticsearch version above 9? Can we use split with '.' sign to use the structure of the version number?